source/getting-started/naming-conventions.md

```diff
@@ -178,10 +178,14 @@ template with the app's instance of `app/controllers/post.js`.
 If you don't specify the controller (`app/controllers/post.js`),
 Ember will automatically make one for you based on the return value
 of the route's `model` hook. If the model is an Array, you get an
-`ArrayController`. Otherwise, you get an `ObjectController`. As of 1.11,
-`ObjectController`s are deprecated and if you try to use its proxy feature you
-will get a warning. Please, take a look at the [deprecations guide](http://emberjs.com/deprecations/v1.x/) for more
-detailed information.
+`ArrayController`. Otherwise, you get an `ObjectController`.
+
+As of 1.11, `ObjectController`s are deprecated and if you try to use its proxy
+feature you will get a warning. Likewise, as of 1.13, `ArrayController`s are
+deprecated.  However, unlike `ObjectController`, using the proxying behavior of
+an auto-generated `ArrayController` will not throw a deprecation message.
```

Comment:
this is also incomplete/incorrect, since `ember-cli` includes an addon that removes the generation of `ObjectController` and `ArrayController`. I would suggest opening a separate PR for this.